test/test-network/systemd-networkd-tests.py

```diff
@@ -578,14 +581,12 @@ def test_geneve(self):
         self.assertRegex(output, 'udp6zerocsumrx')
 
     def test_ipip_tunnel(self):
-        self.copy_unit_to_networkd_unit_path('12-dummy.netdev', '25-ipip-tunnel.netdev', 'ipip.network',
-                                             '25-ipip-tunnel-local-any.netdev', '25-ipip-tunnel-remote-any.netdev')
-        self.start_networkd()
-
-        self.assertTrue(self.link_exits('dummy98'))
-        self.assertTrue(self.link_exits('ipiptun99'))
-        self.assertTrue(self.link_exits('ipiptun98'))
-        self.assertTrue(self.link_exits('ipiptun97'))
```

Comment:
Why remove the assert ?

Reply:
If `wait_online()` passes then the checks are redundant.

Reply:
hmm is not that looking for   ```dummy98:degraded'``` that does not mean link not created ?

Reply:
No. operstate 'degraded' means that the link exists and has only link local addresses, in this case, an ipv6ll address.

Reply:
right thanks.